### PR TITLE
[stable/kube-state-metrics] Version bump to 1.9.0

### PR DESCRIPTION
**Pull request by mrueg**
#### What this PR does / why we need it:
Updates to K-S-M 1.9.0

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/kube-state-metrics/templates/deployment.yaml [hunk 1]
```diff
@@ -110,9 +116,15 @@ spec:
...
         - --collectors=storageclasses
 {{  end  }}
+{{  if .Values.collectors.validatingwebhookconfigurations  }}
+        - --collectors=validatingwebhookconfigurations
+{{  end  }}
 {{  if .Values.collectors.verticalpodautoscalers  }}
         - --collectors=verticalpodautoscalers
 {{  end  }}
+{{  if .Values.collectors.volumeattachements  }}
+        - --collectors=volumeattachements
```
Comment (mbarrien) [inline stable/kube-state-metrics/templates/deployment.yaml:126]: Typo here, extra "e" in "volumeattachEments'. Chart is broken if `volumeattachements` set to true.

Comment (tariq1890) [reply stable/kube-state-metrics/templates/deployment.yaml:126]: Thanks for pointing this out @mbarrien. Would you like to raise a PR to fix this?

**Pull request merged**